yocto/init_ws_ids.sh

```diff
@@ -87,5 +100,6 @@ if [ "${DEVELOPMENT_BUILD}" == "n" ]; then
 	echo "### RELEASE_BUILD ###"
 else
 	echo "### DEVELOPMENT_BUILD ###"
+	echo 'DISTRO_FEATURES:append = " minidebuginfo"' >> ${BUILD_DIR}/conf/local.conf
```

Comment:
can we perhaps put this in the local.conf template and make it conditional like EXTRA_IMAGE_FEATURES?

See: https://github.com/gyroidos/meta-trustx/blob/kirkstone/conf/templates/default/local.conf.sample

Reply:
removed this as discussed